.github/CODEOWNERS

```diff
@@ -31,3 +31,21 @@ Dockerfile @crossedfall
 
 /code/game/gamemodes/dynamic @itsmeow
 /code/modules/modular_computers @itsmeow
+/tools/MapTileAggregator.py @itsmeow
+/code/modules/admin/verbs/_help.dm @itsmeow
+/code/modules/admin/verbs/adminhelp.dm @itsmeow
+/code/modules/admin/verbs/mentorhelp.dm @itsmeow
```

Comment:
Eeehhh. You did do the most recent big changes, but, I don't know if there is any reason to have you specifically own these files.

Reply:
@PowerfulBacon thoughts?

Reply:
Personally I don't see any huge reason to have it specifically assigned to one person (assuming we respect the code ownership) due to the amount of work a lot of different people have put into this system and it still being mostly similar to the original system that we have all worked with before. But at the same time I also don't particularly see any reason against it since we can just ignore code owner requests, but then that kind of defeats the point of it since then you might as well just watch the PR list and check which ones modify an admin file.
For more contributed files like this (where the bulk of it has been made by a large number of different people), I'm not too sure since if we just give code ownership anytime someone does some changes to it then we'd have the entire codebase on code review when every maintainer is equally good at reviewing it, which defeats the purpose of having code owners in the first place since now we are just ignoring them entirely.

Reply:
Yeah, I view code ownership as either being, "This person is the defacto subject-matter expert and really any changes should go through them because it would take more effort for anyone else to review." OR if someone specific should approve of changes before they get deployed in some way (e.g. me and configs)

Stuff that is fairly general and doesn't require explicit approval shouldn't have code ownership.